src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -56,30 +48,30 @@ export const selectStakingDataByAccountSpecifier = createSelector(
   }
 )
 
-export const selectDelegationCryptoAmountByDenom = createSelector(
+export const selectDelegationCryptoAmountByAssetId = createSelector(
   selectStakingDataByAccountSpecifier,
   selectValidatorAddress,
-  selectDenom,
-  (stakingData, validatorAddress, denom): string | undefined => {
+  selectAssetId,
```

Comment:
@elmutt what are we thinking about this? IMO for now we can just go with selecting the first delegation matching `validatorAddress`, no need for `assetId`

Reply:
Yeah I'm happy either way, I think there was a discrepancy in how we envisaged "make web unaware of denoms" but both go in the same direction. That's actually a great option to be ready for future multiple assets in staking/rewards, I'm in